### PR TITLE
feat(library): added CreatePlaylistItemBulkRequest

### PR DESCRIPTION
**Pull request by skyborgff**

### CODE REVIEW

**Comment by Asuka-Langley-Soryu** [comment]
Please align with https://github.com/googleapis/googleapis/blob/master/google/cloud/vision/v1/image_annotator.proto#L74 in the message / method naming and structure.

File: animeshon/library/v1alpha1/library.proto [hunk 1]
```diff
@@ -236,6 +252,14 @@ message CreatePlaylistItemRequest {
   PlaylistItem item = 2 [(google.api.field_behavior) = REQUIRED];
 }
 
+message BatchCreatePlaylistItemsRequest {
+  // The parent this playlist item belongs to.
+  string parent = 1 [(google.api.field_behavior) = REQUIRED];
+
+  // The playlist item to create.
+  repeated PlaylistItem item = 2 [(google.api.field_behavior) = REQUIRED];
```
Comment (Asuka-Langley-Soryu) [inline animeshon/library/v1alpha1/library.proto:260]:
```proto3
  repeated CreatePlaylistItemsRequest requests = 1
      [(google.api.field_behavior) = REQUIRED];
```

File: animeshon/library/v1alpha1/library.proto [hunk 2]
```diff
@@ -130,6 +141,11 @@ message Playlist {
   Type type = 3 [(google.api.field_behavior) = REQUIRED];
 }
 
+message BatchPlaylistItemsResponse {
+  // The id of the playlist.
+  repeated PlaylistItem items = 1 [(google.api.field_behavior) = REQUIRED];
```
Comment (Asuka-Langley-Soryu) [inline animeshon/library/v1alpha1/library.proto:146]:
```
  repeated CreatePlaylistItemResponse responses = 1;
```

File: animeshon/library/v1alpha1/library.proto [hunk 3]
```diff
@@ -130,6 +141,11 @@ message Playlist {
   Type type = 3 [(google.api.field_behavior) = REQUIRED];
 }
 
+message BatchPlaylistItemsResponse {
```
Comment (Asuka-Langley-Soryu) [inline animeshon/library/v1alpha1/library.proto:144]: CreateBatchPlaylistItemsResponse

File: animeshon/library/v1alpha1/library.proto [hunk 4]
```diff
@@ -91,6 +91,17 @@ service Library {
     };
   }
 
+  rpc BatchCreatePlaylistItems(BatchCreatePlaylistItemsRequest) returns (BatchPlaylistItemsResponse) {
+    option (google.api.http) = {
+      post: "/v1alpha1/{parent=users/*/playlists/*}/playlists:createBatch"
```
Comment (Asuka-Langley-Soryu) [inline animeshon/library/v1alpha1/library.proto:96]: batchCreate

File: animeshon/library/v1alpha1/library.proto [hunk 5]
```diff
@@ -91,6 +91,17 @@ service Library {
...
+      body: "*"
+      additional_bindings {
+        post: "/v1alpha1/{parent=audiences/*/playlists/*}/playlists:createBatch"
```
Comment (Asuka-Langley-Soryu) [inline animeshon/library/v1alpha1/library.proto:99]: batchCreate

File: animeshon/library/v1alpha1/library.proto [hunk 6]
```diff
@@ -130,6 +141,11 @@ message Playlist {
   Type type = 3 [(google.api.field_behavior) = REQUIRED];
 }
 
+message BatchPlaylistItemsResponse {
+  // The id of the playlist.
```
Comment (Asuka-Langley-Soryu) [inline animeshon/library/v1alpha1/library.proto:145]: comment is wroooong

File: animeshon/library/v1alpha1/library.proto (hunk 1 above)
Comment (Asuka-Langley-Soryu) [inline animeshon/library/v1alpha1/library.proto:260]: `items`

**Review by christian-roggia — APPROVED** [review]

**Pull request merged**